flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/plan/stream/table/validation/CorrelateValidationTest.scala

```diff
@@ -68,10 +68,6 @@ class CorrelateValidationTest extends TableTestBase {
     //========= throw exception when the called function is a scalar function ====
     util.addFunction("func0", Func0)
 
-    // Java Table API call
-    expectExceptionThrown(
```

Comment:
Why is this not thrown anymore?

Reply:
Because we removed the restriction that it must be a `TABLE` function from the API. A table function is not supported for lateral join yet but it fails at a later stage.